src/index.d.ts

```diff
@@ -15,6 +15,7 @@ interface PlayroomConfig {
   iframeSandbox?: string;
   // eslint-disable-next-line @typescript-eslint/consistent-type-imports
   reactDocgenTypescriptConfig?: import('react-docgen-typescript').ParserOptions;
+  production?: boolean;
```

Comment:
I don't think this is a thing.
```suggestion
```

Reply:
It's specified in the `start` and `build` scripts.

https://github.com/seek-oss/playroom/blob/26828a5a64f7774ed63b60222ba1f6eabe7dbaab/lib/start.js#L10
https://github.com/seek-oss/playroom/blob/26828a5a64f7774ed63b60222ba1f6eabe7dbaab/lib/build.js#L6

Reply:
The first argument is the playroom config. The second argument is not documented/typed anywhere.

Reply:
I don't see where these other options are used.

https://github.com/seek-oss/playroom/blob/a7e478dcc0d5e38f29a04bbf126188c4c2647bfb/lib/start.js#L11-L16

https://github.com/seek-oss/playroom/blob/a04486453b7746a1cb95749e0d41db0e152119e6/lib/makeWebpackConfig.js#L18

Reply:
Ah, good point.